README.md

```diff
@@ -73,7 +73,7 @@ From v0.8.1 you can trigger external tool, for example curl, to send notificatio
 
 ```toml
 [[trigger]]
-cmd = "$HOME/bin/tel.sh"
+cmd = "$HOME/.local/bin/tel.sh"
```

Comment:
Actually, what is this `tel.sh` file?

Reply:
tel.sh is very simple bash script with curl POST alerts into my own telegram channel.

Very similar to the first trigger's cmd in README.md